application/models/User.php

```diff
@@ -694,17 +694,30 @@ public function getManagementColums()
             ],
         ];
 
-        if (Permission::model()->hasGlobalPermission('superadmin', 'read')) {
+        $permission_read_superadmin = Permission::model()->hasGlobalPermission('superadmin', 'read');
+        $permission_read_users      = Permission::model()->hasGlobalPermission('users', 'read');
+        $permission_read_surveys    = Permission::model()->hasGlobalPermission('surveys', 'read');
+
+        // Number of Surveys
+        if ($permission_read_superadmin || ($permission_read_users && $permission_read_surveys)) {
```

Comment:
`$permission_read_superadmin` not needed (superdamin have all other permission)

Reply:
Not only when save, but when test any permission : https://github.com/LimeSurvey/LimeSurvey/blob/9b30c1a73873bc4331a40dad62ed41cf407f5225/application/models/Permission.php#L572

The 1st test done was superadmin read permision (except for superadmin permission)

Reply:
Uh! Thank you for this context. So I think we resolved this thread now.